packages/react-charts/src/components/ChartLabel/ChartLabel.tsx

```diff
@@ -246,7 +245,7 @@ export const ChartLabel: React.FunctionComponent<ChartLabelProps> = ({
     );
   const newStyle = Array.isArray(style) ? style.map(applyDefaultStyle) : applyDefaultStyle(style);
 
-  return <VictoryLabel style={newStyle as any} textAnchor={textAnchor} {...rest} id={id} />;
+  return <VictoryLabel style={newStyle as any} textAnchor={textAnchor} {...rest} {...(id && { id })} />;
```

Comment:
Is it necessary to include id after the rest spread? The `id` prop has already been destructured, so `...rest` should not include it.

Reply:
ah -- that's true

Reply:
We probably doesn't need to destructure `id` at all, considering it's not used anywhere in `ChartLabel`. The `...rest` spread should add the `id` prop.

Reply:
doesn't really matter at this point, but does id need to be pulled out at all anymore?